examples/resources/genesyscloud_flow/resource.tf

```diff
@@ -1,5 +1,6 @@
 resource "genesyscloud_flow" "flow" {
-  filepath = "the flow configuration file path or URL"
+  filepath          = "the flow configuration file path"
+  file_content_hash = filesha256("the flow configuration file path")
```

Comment:
This would have to be done by the terraform file author.  Normally, if I were writing the HCL, I would define a Terraform variable and then just reuse the value